docs/start/getting-started/fragments/flutter/add-api.md

```diff
@@ -1,65 +1,143 @@
-### Setup AWS Cloud Resources with Amplify CLI 
+## Deploy your Amplify sandbox backend
```

Comment:
I think this is the moment we could introduce the "create AWS account". My hypothesis is that at this point the customer has already seen stuff working, they're likely enjoying the smooth experience so far, with very little friction and are more willing to spend time creating an account and provisioning cloud resources, both time consuming tasks. @renebrandel